webclient/components/1time-Mrow-mField/diagnosis/edit-layer/add-ct.vue

```diff
@@ -0,0 +1,297 @@
+<!-- Reference implementation -->
+<!-- For design see webclient/cts/non-temporal/crud/forms.md -->
+<template>
+  <div>
+    <!-- Goal: Show multiple add rows along with remove each row. At end A. Reset B. Add more C. Reviewed -->
+    <el-form>
+      <div v-if="cfGetClientSideTableNewRowsInEditState.length">
+        <el-form-item
+          v-for="ormRow in cfGetClientSideTableNewRowsInEditState"
+          :key="ormRow.clientSideUniqRowId"
+          :class="mfGetCssClassNameForEachDataRow(ormRow.clientSideUniqRowId)"
+        >
+          <!-- Prop explaination  Read prop explanation for span=4 on line 19 -->
+          <el-col :span="20" :class="ormRow.validationClass">
+            <el-select 
+              ref="masterDiagnosisId"
+              :value="mfGetFldValue(ormRow.clientSideUniqRowId, 'masterDiagnosisId')"
+              @input="mfSetFldValueUsingCache($event, ormRow.clientSideUniqRowId, 'masterDiagnosisId')"
+              style="width: 100%"
+              filterable 
+              placeholder="Select diagnosis"
+              @change="fnChangeDiagnosis(ormRow.masterDiagnosisId, ormRow.clientSideUniqRowId)">
+              <el-option
+                v-for="item in cfGetMasterListOfDiagnosis()"
```

Comment:
Line no:24 -> cfGetMasterListOfDiagnosis should be computed function instead of method

Reply:
Rename the function